src/main/java/org/osiam/addons/administration/controller/user/UserViewController.java

```diff
@@ -203,17 +207,101 @@ public String handleLimitAction(@RequestParam(value = REQUEST_PARAMETER_LIMIT) I
 				.build();
 	}
 
-	@RequestMapping(params = REQUEST_PARAMETER_ACTION + "=deactivate")
+	@RequestMapping(params = REQUEST_PARAMETER_ACTION + "=deactivateSelected",method = RequestMethod.POST)
+	public String handlerUserDeactivationSelected (
+			@RequestParam(value = REQUEST_PARAMETER_USER_ID_ARRAY) final String[] id_array,
+			@RequestParam(value = REQUEST_PARAMETER_SEND_MAIL) final Boolean sendMail,
+			HttpServletRequest request) {
+
+		User user;
+		
+		for(String user_id : id_array){
```

Comment:
Code Style: In Java, variables are named lowerCamelCase, i.e. `idArray` instead of `id_array`. Please, fix this for the other lines too.